Debugging.md

```diff
@@ -0,0 +1,70 @@
+# Debugging your Game Servers
+
+Sometimes your game servers cannot start and the Builds are showing up as Unhealthy. In this case, you'd like to see debug your game server running on PlayFab Multiplayer Servers Virtual Machine. Or, sometimes, even though your game server is running smoothly, you may need to connect to the Virtual Machine (VM) and see various statistics/logs about your game server.
```

Comment:
A bit more conversational than I would go with for this doc.

Maybe something like "Many debugging tasks of PlayFab Multiplayer Servers can be done locally using Mock VM Agent. However, sometimes the local environment does not exactly match the actual service and you would like to connect directly to debug a running server because it is marked as unhealthy or not performing as expected."